src/redist/project.json

```diff
@@ -0,0 +1,33 @@
+{
+  "version": "1.0.0-featmsbuild-*",
+  "buildOptions": {
+    "emitEntryPoint": true
+  },
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "type": "platform",
+      "version": "1.0.0"
+    },
+    "tool_csc": "1.0.0-featmsbuild-*",
+    "dotnet": "1.0.0-featmsbuild-*",
+    "tool_nuget": "1.0.0-featmsbuild-*",
+    "tool_msbuild": "1.0.0-featmsbuild-*",
+
+    "MSBuild": "0.1.0-preview-00029-160805",
```

Comment:
Shouldn't all these MSBuild dependencies come through the "tool_msbuild" dependency?  Why duplicate them here?